benchmarks/poppler_pdf_fuzzer/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder@sha256:3a3163b23feb78bc7fc7e27ef42c9e70d977e4e9557667fb6b314790e22f0ccc
```

Comment:
can you please remove the
@sha256:3a3163b23feb78bc7fc7e27ef42c9e70d977e4e9557667fb6b314790e22f0ccc
this is not needed and causes problems with coverage tools.

Reply:
Sorry @inferno-chromium, are you sure about that? The line is added automatically by the oss_fuzz_benchmark_integration.py script. Is that not needed anymore as the commit date is gathered by benchmark.yaml or something? Should we remove that on all future PRs too?

Reply:
@inferno-chromium, I have removed this @sha part, but now instead of 14 bugs reproducing, the binaries only crash on 7 of the inputs.

Reply:
There's 1 bug that's a bit finnicky, that's why we have the 13-14 figure for bugs. Anyway, without the sha, we can't repro half of the bugs anymore. The crash keys become weird.
![image](https://user-images.githubusercontent.com/60978166/111546527-b99b1480-876f-11eb-903c-e7401cd54fb7.png)
![image](https://user-images.githubusercontent.com/60978166/111546584-d0416b80-876f-11eb-8faa-3e810fe63ef9.png)
The first image shows the crash keys for our bugs without the sha part, while the second images shows the crash keys with the sha. What should we do?

Reply:
Do you this issue with any other already integrated bug benchmarks, we removed all shas some time and they are all on master.

Reply:
I am not sure honestly, since I didn't notice the shas were removed on master. I'll check later and I'll let you know, but in general I would expect there to be problems, since we are using 2 different versions of the builder image with and without Sha, right? So programs that had an issue when compiled a certain way, maybe don't have that issue anymore when compiled in another? 
I will report back on what I find later.

Reply:
@inferno-chromium sorry for the late reply. Anyway, I used do_crashes_run on the binaries built with build_coverage for stb_stbi_read_fuzzer, php_php-fuzz-parser-2020-07-25, openh264_decoder_fuzzer, php_php-fuzz-execute, and libhevc_hevc_dec_fuzzer, using as inputs the files in their respective testcases folder.
I have attached a screenshot with the results, but for readability:


| **BENCHMARK** | **BUGS_WITHOUT_SHA** | **BUGS_WITH_SHA** |
| --- | --- | --- |
| stb_stbi_read_fuzzer | 5/11 | 8/11 |
| php_php-fuzz-parser-2020-07-25 | 3/8 | 6/8 |
| openh264_decoder_fuzzer | 1/12 | 4/12 |
| php_php-fuzz-execute | 8/9 | 9/9 |
| libhevc_hevc_dec_fuzzer | 1/12 | 11/12 |

As you can see, the number of bugs that reproduces is always larger when we use the sha (which I think makes sense, since these bugs were originally found with that version of the base-builder, right?).

I am not sure why the removal of the sha was needed, I only saw that it caused problems with a fuzzer in an issue, but if we don't put the sha in the Dockerfile, a lot of bugs end up not triggering anymore, which is not what we want.

![image](https://user-images.githubusercontent.com/60978166/112194125-7a057a00-8c00-11eb-87f7-b826d07f774d.png)